src/app/shell/workshop-details/side-menu/actions/actions.component.ts

```diff
@@ -11,7 +12,7 @@ import { Role } from 'src/app/shared/enum/role';
   styleUrls: ['./actions.component.scss']
 })
 export class ActionsComponent implements OnInit {
-  displayed: boolean = this.isDisplayed();
+  displayed: boolean = this.isRegistered();
```

Comment:
change the naming of the variable to isRegistered, so it will match with the related action/function. Also, use approach for such variable with is/has naming: isDisplayed, isRegistered